api/openapi.yaml

```diff
@@ -145,49 +156,438 @@ paths:
+          description: Bad Request
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        default:
+          $ref: '#/components/responses/DefaultResponse'
+      description: Get all the information about a specific place in a specific organization.
+      tags:
+        - stalker web app
+  '/organization/{organizationId}/places':
+    parameters:
+      - $ref: '#/components/parameters/organizationId'
+    post:
+      description: Add a new place to the given organization. Operation reserved only to the managers and up.
+      operationId: postPlaceByOrganizationId
+      parameters:
+        - $ref: '#/components/parameters/organizationId'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: ./model/PlaceData.v1.yaml
+        required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: ./model/GenericId.v1.yaml
+          description: Ok. The response contains the id of the newly created place.
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        default:
+          $ref: '#/components/responses/DefaultResponse'
+      summary: Create a new place.
+      tags:
+        - stalker web app
+    get:
+      summary: Read places of the organization.
+      operationId: get-organization-organizationId-places
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  places:
+                    $ref: ./model/PlaceData.v1.yaml
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        default:
+          $ref: '#/components/responses/DefaultResponse'
+      description: Get all the places of a specific organization.
+      tags:
+        - stalker mobile app
+        - stalker web app
+  '/organization/{organizationId}/report':
+    parameters:
+      - $ref: '#/components/parameters/organizationId'
+    get:
+      summary: Read an organizational report.
+      tags:
+        - stalker web app
+      responses:
+        '204':
+          description: Not implemented yet.
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '501':
+          description: Unfinished endpoint.
+        default:
+          $ref: '#/components/responses/DefaultResponse'
+      operationId: get-organization-organizationId-report
+      description: 'Get a report detailing user accesses, time spent at each place and most frequented places. Optional feature.'
+  '/organization/{organizationId}/user/{userId}':
```

Comment:
serve questo endpoint se abbiamo già user/{userId}?

Reply:
no in effetti è ridondante. lo tolgo!